<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Voter</class>
 <widget class="QWidget" name="Voter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>387</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>91</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Place:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Time:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QTimeEdit" name="timeEdit">
     <property name="displayFormat">
      <string>HH:mm</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Vote</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Place</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Time</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string># Votes</string>
      </property>
      <property name="textAlignment">
       <set>AlignRight|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Vote</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Voter</receiver>
   <slot>vote_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>Voter</receiver>
   <slot>tablevote_clicked(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal1(int)</signal>
  <slot>vote_clicked()</slot>
  <slot>tablevote_clicked(int,int)</slot>
 </slots>
</ui>
